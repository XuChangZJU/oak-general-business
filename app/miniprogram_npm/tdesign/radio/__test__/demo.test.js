import simulate from 'miniprogram-simulate';
import getDemoPath from '../../../test/utils/getDemoPath';

describe('radio basic demo', () => {
  let id;
  beforeAll(() => {
    id = simulate.load(getDemoPath('radio', 'basic'), {
      less: true,
    });
  });
  test('demo should render correctly', async () => {
    const container = simulate.render(id);
    container.attach(document.createElement('parent-wrapper'));
    expect(container.toJSON()).toMatchSnapshot();
  });
});
