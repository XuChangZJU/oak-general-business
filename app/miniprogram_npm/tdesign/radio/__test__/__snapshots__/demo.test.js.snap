// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`radio basic demo demo should render correctly 1`] = `
Object {
  "attrs": Array [],
  "children": Array [
    Object {
      "attrs": Array [],
      "children": Array [
        Object {
          "attrs": Array [
            Object {
              "name": "class",
              "value": "basic-radio",
            },
            Object {
              "name": "name",
              "value": "radio1",
            },
            Object {
              "name": "title",
              "value": "单行标题",
            },
          ],
          "children": Array [],
          "event": Object {},
          "tagName": "t-radio",
        },
        Object {
          "attrs": Array [
            Object {
              "name": "name",
              "value": "radio2",
            },
            Object {
              "name": "title",
              "value": "双行标题，长文本自动换行，该选项的描述是一段很长的内容",
            },
          ],
          "children": Array [],
          "event": Object {},
          "tagName": "t-radio",
        },
      ],
      "event": Object {},
      "tagName": "wx-view",
    },
  ],
  "event": Object {},
  "tagName": "main",
}
`;
